Remove commit message label on docker container

Docker build fails when the following commit message is used.

```
[DOCKER-322] Change tags when creating a build for a test branch

On Jenkins, all builds are in one namespace. We don't want branches to write to the 'official' images on Jenkins,
because then other builds will pick up that image from a branch build and use it.
For all branches other than master & release, append a '-build-$branchName-$buildNumber' identifier
```

This is related to https://github.com/xenit-eu/alfresco-docker-gradle-plugin/blob/cddb53f3ac45708805f8c28fde7734dfae1d5df4/src/main/java/eu/xenit/gradle/docker/DockerBuildBehavior.java#L124-L126, where the commit message is added to a docker label.

It fails the build with

```
> Task :1skeleton:4.2:buildDockerImage FAILED

:1skeleton:4.2:buildDockerImage (Thread[Execution worker for ':',5,main]) completed. Took 1 mins 26.192 secs.

FAILURE: Build failed with an exception.

* What went wrong:

Execution failed for task ':1skeleton:4.2:buildDockerImage'.

> Could not build image: failed to process "'\"[DOCKER-322] Change tags when creating a build for a test branch\\\n\\\nOn Jenkins, all builds are in one namespace. We don't want branches to write to the 'official' images on Jenkins,\\\nbecause then other builds will pick up that image from a branch build and use it.\\\nFor all branches other than master & release, append a '-build-$branchName-$buildNumber' identifier\"'": unexpected end of statement while looking for matching double-quote
```

https://travis-ci.com/github/xenit-eu/docker-alfresco/jobs/329205214#L1913-L1920

--- a/src/main/java/eu/xenit/gradle/docker/DockerBuildBehavior.java
+++ b/src/main/java/eu/xenit/gradle/docker/DockerBuildBehavior.java
@@ -121,9 +121,6 @@ public class DockerBuildBehavior {
             labels.put(labelPrefix + "branch", gitInfoProvider.getBranch());
             labels.put(labelPrefix + "commit.id", gitInfoProvider.getCommitChecksum());
             labels.put(labelPrefix + "commit.author", gitInfoProvider.getCommitAuthor());
-            labels.put(labelPrefix + "commit.message", '"' + gitInfoProvider.getCommitMessage()
-                    .replaceAll("\"", "\\\\\"")
-                    .replaceAll("(\r)*\n", "\\\\\n") + '"');
         }
         return labels;
     }

